playwright/README.md

```diff
@@ -0,0 +1,68 @@
+# Playwright Testing ReadMe
+
+A small readMe on how to run the playwright suite and the commands required
+
+## Table of Contents
+
+* [How to set up throttle limiter for testing](#Throttler-Setup)  
+* [How to run playwright](#Running-Playwright)
+    * [Running all tests](#running-all-tests)  
+    * [Running a single test](#running-a-single-test)
+    * [Run a set of tests](#run-a-set-of-test-files)  
+    * [Running tests in heeaded mode](#running-tests-in-headed-mode)
+    * [Running tests on a specific browser engine](#running-tests-on-specific-browser)
+
+
+## How to set up throttle limiter for testing
+
+When building the test suite it was found that the tests would max out the rate limiter. A change was made to make this a variable which can be set to different values. 
+
+To set this up for testing do the following:
+
+* Make sure you have down docker compose down 
+* Before building a new docker image go into you `.env` and set the `RATE_LIMIT` variable to **1000** (this is a suggestion if you think you need more put more, if you think you need less put less)
+* Build and run a new docker image via docker-compose build and docker-compose up
+
+This is to be done before any tests are run otherwise the limiter will max out and the tests will fail.
+
+## How to run playwright
+
+By default tests will be run on all 3 browsers, chromium, firefox and webkit using 3 workers. This can be configured in the `playwright.config` file. Tests are run in headless mode meaning no browser will open up when running the tests. Results of the tests and test logs will be shown in the terminal.
+
+```
+cd playwright/
+```
+
+### Running all tests:
+
+```
+npx playwright test
+```
+
+### Running a single test:
+
+Navigate into test folder you would like to run within the playwright folder i.e /ui-tests.
+```
+npx playwright test file-name
+```
+
+### Run a set of test files:
+
+```
+npx playwright test ui-tests/todo-page/ ui-tests/landing-page/
```

Comment:
Would it be better to make this example an actual test file that we have in the repo?

Reply:
Can do, this was the example off the docs